config/webhook/webhook.go

```diff
@@ -33,14 +37,54 @@ func main() {
 }
 
 type EvictionHandler struct {
+	Client  client.Client
 	decoder *admission.Decoder
 }
 
 func (e *EvictionHandler) Handle(ctx context.Context, req admission.Request) admission.Response {
 	logger.Info("Received eviction request", "namespace", req.Namespace, "name", req.Name)
 
 	// Log eviction request
-	fmt.Printf("Eviction requested for Pod %s/%s\n", req.Namespace, req.Name)
+	evictionLog := myappsv1.EvictionLog{
+		PodName:      req.Name,
+		EvictionTime: time.Now().Format(time.RFC3339),
+	}
+
+	// List all PDBWatchers
+	pdbWatcherList := &myappsv1.PDBWatcherList{}
+	err := e.Client.List(ctx, pdbWatcherList, &client.ListOptions{Namespace: req.Namespace})
```

Comment:
is this client using the informer cache? If not this could be tremenduously expensive to do for every eviction with even a modest amount of pdbs